front/app/containers/ProjectsShowPage/index.tsx

```diff
@@ -120,6 +154,24 @@ const ProjectsShowPage = memo<Props>(({ project, scrollToEventId }) => {
   } else if (projectNotFound) {
     content = <ProjectNotFound />;
   } else if (projectId && processType) {
+    let phaseParticipationMethod;
+
+    if (!isNilOrError(phases)) {
+      const phase = phaseIdUrl ? getPhase(phaseIdUrl, phases) : null;
+      if (!isNilOrError(phase)) {
```

Comment:
Minor, we could reduced 161 to 163 to 

```
      phaseParticipationMethod = phase?.attributes.participation_method || getCurrentPhase(phases)?.attributes.participation_method;
```

And then handle the case where there is no participation method since it could be null based on the flow. With `any` type above, typescript doesn't warn us about this case.